README.md

```diff
@@ -21,11 +21,12 @@ Add the module to your Terraform resources:
 
 ```hcl
 module "rds-postgres-password" {
-  source    = "github.com/gearnode/terraform-kubernetes-get-secret?ref=v0.2.0"
+  source    = "github.com/gearnode/terraform-kubernetes-get-secret?ref=v0.3.0"
 
   namespace = "default"
   name = "terraform"
   key = "database-password"
+  context = "supercontext"
```

Comment:
```suggestion
  context = "k8s-context"
```

Reply:
I don't have write permissions for this suggestion. let merge this one then create one to fix what we need to fix or maybe we should have created an upstream branch but this will starting to be to complicated